features/03_testing_frameworks/cucumber/steps/command/debug_your_command_in_aruba.feature

```diff
@@ -53,11 +50,15 @@ Feature: Debug your command in cucumber-test-run
     And I type "foo"
     And I type "exit"
     Then the output should contain:
-    """
-    [1] pry(main)> foo
-    => "hello"
-    [2] pry(main)> exit
-    """
+      """
+       => 8: binding.irb
+          9: 
+          10: exit 0
+      Switch to inspect mode.
```

Comment:
This looks weird.

Reply:
There doesn't seem to be much we can do about this. Maybe we should keep `pry` around for now and just drop `byebug`.